sloglint.go

```diff
@@ -226,6 +228,14 @@ func visit(pass *analysis.Pass, opts *Options, node ast.Node, stack []ast.Node)
 	// NOTE: "With" functions have no message argument and must be skipped.
 	if opts.StaticMsg && msgPos >= 0 && !isStaticMsg(call.Args[msgPos]) {
 		pass.Reportf(call.Pos(), "message should be a string literal or a constant")
+
+	}
+
+	if opts.LowercaseMsg && msgPos >= 0 {
+		msg := getBasicLitValue(call.Args[msgPos])
+		if len(msg) > 2 && unicode.IsUpper(rune(msg[1])) {
```

Comment:
You should/could explain why you are only taking msg[1], and not msg[0]

I assume it's because 0 might be double quote or backtick